RealLifeGTAV/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "author": {
+    "name": "N0chteil ^^",
+    "id": "495901098926669825"
+  },
+  "service": "Real Life GTAV",
+  "description": {
+    "en": "Real Life GTAV is a german FiveM roleplay server with custom scripts and more!",
+	"de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
```

Comment:
fix this tab too.

Reply:
still not fixed tabs.

Reply:
I see no difference between the current one.

Reply:
```	"de":``` there are 2 tabs instead of one like before ```"en":```